packages/liveblocks-yjs/src/index.ts

```diff
@@ -0,0 +1,253 @@
+// TODO: apparently YJS is full of anys or something, see if we can fix this
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import type {
+  BaseUserMeta,
+  Json,
+  JsonObject,
+  LsonObject,
+  Room,
+} from "@liveblocks/client";
+import { Base64 } from "js-base64";
+import { Observable } from "lib0/observable";
+import * as Y from "yjs";
+
+type RoomEvent = {
+  type: "REFRESH";
+};
+
+type LiveblocksYjsOptions = {
+  httpEndpoint?: string;
+};
+
+type RefreshResponse = {
+  updates: string[];
+  lastUpdate: number;
+};
+
+type MetaClientState = {
+  clock: number;
+  lastUpdated: number;
+};
+
+/**
+ * This class will store YJS awareness in Liveblock's presence under the __yjs key
+ * IMPORTANT: The yjs awareness protocol uses ydoc.clientId to reference users
+ * to their respective documents. To avoid mapping yjs clientIds to liveblock's connectionId,
+ * we simply set the clientId of the doc to the connectionId. Then no further mapping is required
+ */
+export class Awareness extends Observable<any> {
```

Comment:
Please avoid using `any`. It disables typechecking and infects surrounding code to be type safe. Most of the time, you can just replace it with `unknown` instead. That will still work, but forces you to validate a value before you can use it, rather than silently allowing anything.

Reply:
I just checked and you should be able to replace all `any` types with `unknown` in this module without issues. ✅